tensorflow/lite/tools/make/build_riscv_lib.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash 
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+set -x
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TENSORFLOW_DIR="${SCRIPT_DIR}/../../../.."
+
+FREE_MEM="$(free -m | awk '/^Mem/ {print $2}')"
+# Use "-j 4" only memory is larger than 2GB
```

Comment:
I needed this to use it on RPI3. I think most desktops are fine to just use "make -j" without giving a job numer?

Reply:
You are right. But on my side I am working on freedom-u540 which has 4 core. It is some what similar to raspberry pi in terms of number of cores. Hence I kept the script same.
For more details I am providing a link on freedom-u540 https://www.sifive.com/boards/hifive-unleashed
Please let us know if you need additional info.

Reply:
@terryheo Can you suggest what is needed from my side.

Reply:
If that's the case, it's fine to keep the logic.